sdk/python/tests/test_client.py

```diff
@@ -448,6 +515,82 @@ def test_apply_feature_table_success(self, test_client):
             and feature_tables[0].entities[0] == "fs1-my-entity-1"
         )
 
+    @pytest.mark.integration
+    @pytest.mark.parametrize(
+        "test_client", [lazy_fixture("client_with_object_registry_gcs")],
+    )
+    def test_apply_feature_table_integration(self, test_client):
+
+        test_client.set_project("project1")
```

Comment:
Is `project1` supposed to be used somewhere?

Reply:
this is copied from `test_apply_feature_table_success`; I checked and it's not used/needed in either